src/data/move.ts

```diff
@@ -2870,6 +2886,9 @@ export class ForceSwitchOutAttr extends MoveEffectAttr {
       if (!player && !user.scene.currentBattle.battleType) {
         if (this.batonPass)
           return false;
+        //U-turn et al should not switch a wild mon out, but a player's Dragon Tail can
+        if (!(user instanceof PlayerPokemon) && move.category!=MoveCategory.STATUS)
```

Comment:
```suggestion
        if (!(user instanceof PlayerPokemon) && move.category !== MoveCategory.STATUS)
```
Minor formatting nitpick

Reply:
This should probably use hasTrainer too right?